libs/zzz/formula-ui/src/char/util.ts

```diff
@@ -9,7 +9,9 @@ export function getVariant(tag: Tag) {
 }
 export function isDmg(tag: Tag) {
   const { q } = tag
-  return ['standardDmg', 'anomalyDmg', 'sheerDmg'].includes(q as string)
+  return ['standardDmg', 'anomalyDmg', 'sheerDmg', 'nonDmg'].includes(
```

Comment:
How confusing that nondmg is considered DMG lol. Should we rename this? Or maybe a better solution (idk one)?